bindings/protocols/bacnet/index.html

```diff
@@ -1048,12 +1080,70 @@ <h2 id="examples">Examples</h2>
 }
 </pre>
 
-<p>
-    [[[#example-enum-mapping]]] shows how an <code>enum</code> in the Data Schema is extended in the <code>forms</code> 
-    to make it understandable by BACnet drivers.
-</p>
+        <p>
+            [[[#example-uri-variable-for-writing]]] demonstrates how the <code>uriVariable</code> in the
+            affordance-level can be used to model a writing priority in a <code>forms</code> element.
+            The <code>"bacv:commandPriority"</code> indicates that the URI variable set by the Consumer application will
+            be relayed as the priority argument in the WriteProperty request.
+            For other operations (readproperty, observeproperty, unobserveproperty) this URI variable is irrelevant, so the corresponding forms do not use the URI variable.
+            If the URI variable is also irrelevant for the writeproperty operation (e.g. in case the BACnet object doesn't have a priority array),
+            it can also be left out of the writeproperty form and the URI variable can be omitted completely.
+            The URI variable comes with a default, so in case the Consumer doesn't provide this URI variable, the default of 16 will be used.
```

Comment:
```suggestion
            If the URI variable is also irrelevant for the writeproperty operation (e.g., when the BACnet object doesn't have a priority array),
            it can also be left out of the writeproperty form and the URI variable can be omitted completely.
            The URI variable comes with a default, so if the Consumer doesn't provide this URI variable, the default of 16 will be used.
```